ci/resources/setup_ltp.sh

```diff
@@ -29,12 +29,12 @@ then
     cp -f manifest_RHEL.template manifest.template
 fi
 
-if [[ "$base_os" == *"20.04"* ]] || [[ "$base_os" == *"22.04"* ]]
+if [[ "$base_os" == *"20.04"* ]] || [[ "$base_os" == *"22.04"* ]] || [[ "$base_os" == *"24.04"* ]]
```

Comment:
Instead of having all os version check for ubuntu, we can check here, if base_os == *ubuntu*, then also it will work,

Reply:
Done